src/calendar/index.ts

```diff
@@ -324,20 +358,24 @@ export class CalendarBase<P extends CalendarProperties = CalendarProperties> ext
 		return weeks;
 	}
 
-	protected renderDateCell(date: number, index: number, selected: boolean, currentMonth: boolean, today: boolean): DNode {
+	protected renderDateCell(dateObj: Date, index: number, selected: boolean, currentMonth: boolean, today: boolean): DNode {
```

Comment:
If someone extends this class and overrides this method, the change from `date` to `dateObj` could be a breaking change.

Reply:
we're moving away from those kind of extensions in favour of custom render properties etc so I shouldn't worry.

Reply:
Why did this need to be changed to a dateObj btw?

Reply:
Okay, great. I figured this was an unlikely method to override anyway.

I switched to a date object because we need to do the min/max. We were already creating the object in the caller of this function but were were just using it for string comparison.